src/components/MyInquiry/MyInquiry.module.scss

```
@@ -18,6 +18,6 @@ $highlight: #ff7f23;
   text-decoration: none;
 }
 
-.move {
+.moveIcon {
```

클래스명의 단어단위 구분은 하이픈으로 통일해 쓰고있어요!
`styles`를 import하는 구조때문에 사실 camelCase로 적고싶지만.. 다른 코드들이 이미 통일돼서 맞추는 수밖에 없는 것 같아요 ㅠ ㅋㅋ
```suggestion
.move-icon {
```

---

억..왜 옛날에 만들었던 문의하기 페이지가 같이 PR에 올라갔는지는 모르겠지만...수정이 필요한 부분이 있었군요..! 수정했습니다!

---

그이유는... 지금 브랜치 (#71) 브랜치가 이전 이슈에 대해 다루고 있기 때문이죠 ㅋㅋ 다음엔 브랜치 새로 만들어서 진행해주세요

---

악,,넵!